behaviors/wow_retail/druid/restoration.lua

```diff
@@ -164,23 +189,33 @@ local function DruidRestoHeal()
     local u = v.Unit
     local prio = v.Priority
 
-    if wildgrowth and Spell.WildGrowth:CastEx(u) then return end
 
+    -- TODO convoke and tranq logic need to take into account Multiple people low
+    if Settings.DruidRestoOvergrowth and u.HealthPct < 25 and Spell.Overgrowth:CastEx(u) then return end
     if u.HealthPct < 50 and Spell.CenarionWard:CastEx(u) then return end
     if u.HealthPct < 50 and (u:HasBuffByMe("Rejuvenation") or u:HasBuffByMe("Regrowth")) and
-        Spell.Swiftmend:CastEx(u, SpellCastExFlags.NoUsable) then return end
+        Spell.Swiftmend:CastEx(u, SpellCastExFlags.NoUsable) then
+      return
+    end
+
+    if Settings.DruidInnervate and Me.PowerPct < 25 and Spell.Innervate:CastEx(Me) then return end
+    if wildgrowth and Spell.WildGrowth:CastEx(u) then return end
+
     -- fix ugly
     if Me.ShapeshiftForm == ShapeshiftForm.Cat then
       if u.HealthPct < 80 and not u:HasBuffByMe("Rejuvenation") and Spell.Rejuvenation:CastEx(u) then return end
     else
       if u.HealthPct < 92 and not u:HasBuffByMe("Rejuvenation") and Spell.Rejuvenation:CastEx(u) then return end
     end
+    if u.HealthPct < 45 and Spell.AdaptiveSwarm:CastEx(u) then return end

```

Comment:
Adaptive swarm already implemented and will cast on the tank if no adaptive swarm is present, move the call from https://github.com/TeamWector/Pallas/blob/eeab8d6d34d57a9a975353269c5d8b62f07f79f8/behaviors/wow_retail/druid/restoration.lua#L244 if you want it to cast on any low health target